products/cli/project-commands/autofix.md

```diff
@@ -0,0 +1,32 @@
+---
+nav:
+  title: Autofixer
+  position: 7
+
+---
+
+# Autofixer
+
+Shopware-CLI comes with some builtin auto fixers for project migrations.
+
+## Migrate a Project to Symfony Flex
+
+Prior to Shopware 6.5, Shopware didn't use Symfony Flex. This means that the project structure was different and some configuration files were located in different places. The `shopware-cli project autofix flex` command will migrate your project to Symfony Flex and move all configuration files to the correct locations.
+
+::: warning
+Ensure that you have a backup of your project before running this command.
+:::
+
+```bash
+shopware-cli project autofix flex
+```
+
+The command will delete all configuration files that are not needed anymore. It will update the `composer.json` file to use the new configuration files and will also update the `bin/console` file to use the new configuration files.
```

Comment:
```suggestion
The command will delete all unnecessary configuration files. It will also update the `composer.json` file and the `bin/console` file to use the new configuration files.
```